packages/client/tests/functional/_utils/setupTestSuiteMatrix.ts

```diff
@@ -78,6 +81,18 @@ function setupTestSuiteMatrix(
         globalThis['Prisma'] = (await global['loaded'])['Prisma']
       })
 
+      // for better dx, copy the first generated client into the test suite root
+      beforeAll(async () => {
+        const rootNodeModuleFolderPath = path.join(suiteMeta.testRoot, 'node_modules')
+
+        const suiteFolderPath = getTestSuiteFolderPath(suiteMeta, suiteConfig)
+        const suiteNodeModuleFolderPath = path.join(suiteFolderPath, 'node_modules')
+
+        await fs.copy(suiteNodeModuleFolderPath, rootNodeModuleFolderPath, { recursive: true }).catch((e) => {
+          if (e.code !== 'EEXIST') throw e
```

Comment:
In which case `EEXIST` will happen with `recursive`? I thought it will always override existing files in this case.

Reply:
Yep, you're right. Wrong assumption from my side. Thanks. My initial intention was that if the `node_modules` folder exists, then it would fail and not try to overwrite. Kinda like having a lockfile, but with a folder so that only one jest job does the copy into `node_modules`. I think I can achieve that with mkdir instead.

Reply:
Fixed via `mkdir` to use `node_modules` as a locking mechanism.